rocketpy/AeroSurfaces.py

```diff
@@ -633,12 +633,25 @@ def rollInfo(self):
                 self.rollForcingInterferenceFactor
             )
         )
-
+        # lacks a title for the plot
         self.rollParameters[0]()
+        # lacks a title for the plot
         self.rollParameters[1]()
 
         return None
 
+    def airfoilPlots(self):
+
+        if self.airfoil is not None:
+            airfoilCl = Function(
+                self.airfoil[0],
+                inputs="angle of attack",
+                outputs="lift coefficient",
+                interpolation="linear",
+                extrapolation=None,
+            )
```

Comment:
It would be better to just change the `airfoilCl` on line 385 to `self.airfoilCl` and just call the plot here with `self.airfoilCl()`. This way we avoid repeating code.